src/core/src/op/sparse_fill_empty_rows.cpp

```diff
@@ -34,6 +34,13 @@ void SparseFillEmptyRows::validate_and_infer_types() {
                           "The element type of the dense_shape input must be i32 or i64. Got: ",
                           dense_shape_element_type);
 
+    NODE_VALIDATION_CHECK(this,
+                          dense_shape_element_type == indices_element_type,
```

Comment:
The both types could be merged  and then merged type can be checked if is i32 or i64

Reply:
Done.